pkg/addons/adminconsole/adminconsole.go

```diff
@@ -53,7 +53,8 @@ var helmValues = map[string]interface{}{
 		"enabled":  true,
 		"nodePort": DEFAULT_ADMIN_CONSOLE_NODE_PORT,
 	},
-	"embeddedClusterID": metrics.ClusterID().String(),
+	"embeddedClusterID":      metrics.ClusterID().String(),
+	"embeddedClusterVersion": defaults.Version,
```

Comment:
IIRC this isn't actually used for anything besides populating a configmap?

Reply:
Is that configmap actually used for something?

Reply:
it's passed as an environment variable to the kotsadm deployment for reporting purposes

Reply:
https://github.com/replicatedhq/kots-helm/pull/35

Reply:
alternatively, we can store the embedded cluster in the cluster somewhere and have KOTS detect it from there. not sure which approach is better.

Reply:
oh gotcha
yeah feel free to fix this then